### DISCUSSION TITLE
Dual mining ETH and TON Very low hasrate on rtx 3080s

### DISCUSSION THREAD

**Discussion by Sibelius25** [Lolliedieb/lolMiner-releases#1236] [General]
Hi guys!

I have just tried to dualmine ETH and Ton. I have 6 RTX 3080 and one rtx 2070 Super. The latter seems to be doing fine, but the RTX 3080s all get mysteriously low hash rate from under 100 MH to about 350 MH. Anyone have any idea how to fix this? 

Would greately appreciate any help on the matter :)

https://user-images.githubusercontent.com/99537651/153709917-72e58b78-a6dc-42a9-a54c-cf9965fa2582.PNG

**Comment by Kostas-13** [comment]
Try raising PL to 300-320w 
Lock gpu vcore at 725-731
Play with gpu clocks 1200-1500mhZ
